services/provider/server/server.go

```diff
@@ -570,7 +570,10 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 				keyProp = "adminKey"
 			}
 			extR = append(extR, &pb.ExternalResource{
-				Name: clientSecretName,
+				// a common suffix '.csi' is being added to distinguish secrets that are created
```

Comment:
The same change might also be required for healthcheck user as well, please check

Reply:
@leelavg putting an hold on the PR to make sure we address that as well

Reply:
@Madhu-1 after creating a claim (while testing the PR) I looked at the secrets that are created in provider & client side but only found these having same name, where do you want me to check for healthcheck user?

Btw, I did check other resources that are part of another RPC (getstorageconfig) as we discussed offline the other day.

Reply:
yes i forgot the other day, Healthcheck Secrets also follows the same patterns check the secret name on the storageclassclaim status and see if its having the same name as the secret name in health checker ceph user.

Reply:
from https://github.com/red-hat-storage/ocs-operator/pull/2512#issuecomment-2007302624 you can see there are only 4 secrets (2 per claim stored in status) and we are renaming those.

Reply:
yes i checked the code we are not using health check any more should we remove the code but one need to check and confirm the same that its not required.

Reply:
we are sending many details to the consumer cluster but that's not used any more please check if that's the case

https://github.com/red-hat-storage/ocs-operator/blob/d8c47891eea0b78e5483e47d399fc81de273c001/services/provider/server/server.go#L273-L368

can be addressed in a different PR.

Reply:
@leelavg @Madhu-1 The health check secret is attached and not associated with any single claim it is associated with the entire client/consumer. It is still in use

Reply:
@nb-ohad AFAIK we are not using the health check secret anymore on the consumer cluster.